framework/doc/content/source/timesteppers/CSVTimeSequenceStepper.md

```diff
@@ -1,18 +1,15 @@
 # CSVTimeSequenceStepper
 
-Imports a time sequence from a CSV file, or similar delimited text files.
-The CSVTimeSequenceStepper uses a [DelimitedFileReader](MooseUtils.md#delimitedfilereader)
+This time stepper derives from [TimeSequenceStepperBase.md] and provides the
```

Comment:
I think being more explicit is better, let's point to what it inherits from that class, as users should not need to understand OOP to use MOOSE